### PR TITLE
Fix @dataProvider example

### PR DESCRIPTION
**Pull request by FallDi**
* annotation is case-sensitive
* dataProvider must be `public`

### CODE REVIEW

File: docs/07-AdvancedUsage.md [hunk 1]
```diff
@@ -249,7 +249,7 @@ You can also use the `@dataprovider` annotation for creating dynamic examples, u
     /**
      * @return array
      */
-    protected function pageProvider() // alternatively, if you want the function to be public, be sure to prefix it with `_`
+    public function pageProvider() // alternatively, if you want the function to be public, be sure to prefix it with `_`
```
Comment (Naktibalda) [inline docs/07-AdvancedUsage.md]: Your change conflicts with comment.

Comment (FallDi) [reply docs/07-AdvancedUsage.md]: @Naktibalda i'm not sure what really comment means. Remove it as outdated

Comment (Naktibalda) [reply docs/07-AdvancedUsage.md]: If you make method public in Cest file, Codeception will run it as a test.

Comment (FallDi) [reply docs/07-AdvancedUsage.md]: Wow, I see.
My mistake is create `Test` file instead of `Cest`.

Sorry for any inconvenience

**Pull request closed without merging**